docs/collections/keyvalue/key-value-store-data.md

```diff
@@ -29,6 +29,10 @@ The key that corresponds to the document. Every key has a unique value.
 
 Displays the value corresponding to the key.
 
+## Group ID
```

Comment:
If no Group ID was given while creating or modifying the document, we will see "N/A", Not sure if it should be mentioned.

Reply:
Good info, doesn't hurt to add it.